client/signup/jetpack-connect/plans-landing.jsx

```diff
@@ -18,15 +18,12 @@ const CALYPSO_JETPACK_CONNECT = '/jetpack/connect';
 class PlansLanding extends Component {
 	static propTypes = {
 		basePlansPath: PropTypes.string,
-		hideFreePlan: PropTypes.bool,
 		intervalType: PropTypes.string,
-		isLanding: PropTypes.bool,
-		landingType: PropTypes.string,
 	};
 
 	componentDidMount() {
 		this.props.recordTracksEvent( 'calypso_jpc_plans_landing_view', {
-			jpc_from: this.props.landingType
+			jpc_from: 'jetpack',
```

Comment:
No, not certain, but I prefer clean this up since it's not in use now rather than leave it lying around. Passing a few props around isn't very complicated to implement if we need. What do you think?